tests/unit/main.fmf

```diff
@@ -5,4 +5,6 @@ test: python3 -m pytest -v
 framework: shell
 require+:
   - python3-pytest
+  - type: file
+    pattern: examples/*
```

Comment:
Shouldn't the pattern start with a slash?

Reply:
doesn't really matter, it could be a problem if someone later adds another examples directory somewhere else in this repo it would also be included, but that would happen with the begging slash anyway